policies/release-policy.md

```diff
@@ -0,0 +1,124 @@
+committee. Upon successful completion of this check, an alpha release is
+created, marking the beginning of the Alpha phase.
+
+### 4. Release
+The Release phase of OpenSSL library commences with the creation of an alpha
+release at the end of the Development phase and spans over a period of six
+weeks. In the Release phase, no new features can be added, nor can any code
+refactoring be planned to be undertaken. Effective communication at this stage
+is crucial to ensure the quality of the release and, consequently, the success.
+
+  * **Alpha**:
+  During this phase, feature development is complete and feedback from the
+  community is being actively addressed. Testing, bug fixing, and documentation
+  updates are ongoing. Any critical issues discovered may lead to the removal of
+  the offending code.
+  * **Beta**:
+  This phase signifies the library is nearing its final release. Testing remains
+  intensive, but the focus is on ensuring stability. While all identified
+  regressions and critical concerns from the alpha phase must have been
+  addressed, only crucial, last-minute fixes are expected here.
+  * **Final**:
+  The Final phase is focused on release preparation. No code fixes, no
+  documentation amendments — just the steps essential for the release. The tasks
+  involved are limited to the necessary steps described in the [Making an
+  OpenSSL Release] document, such as updating the list of known issues, tagging
+  the code in the git tree, and issuing digital signatures. Once these tasks are
+  completed, the phase culminates with the new release being made publicly
+  available and announced.
+
+See [Release Requirements Policy] for details on each type of release.
+
+### 5. Support
+  * **Full Support**:
+  Upon the final release a one-year Full Support period is initiated for regular
+  releases, and a four-year Full Support period for LTS releases. During this
+  phase, bugs and security issues are addressed and fixed according to the
+  [Stable Release Updates Policy].
+  * **Maintenance Support**:
+  Immediately after the Full Support phase ends, the Maintenance Support phase
+  begins, lasting for one year. During this phase, the primary focus is on
+  fixing security issues, although other bugs may be addressed at the discretion
+  of OpenSSL engineering.
+
+[Stable Release Updates Policy]: https://www.openssl.org/policies/technical/stable-release-updates.html
+[Release Requirements Policy]: https://www.openssl.org/policies/technical/release-requirements.html
+[Making an OpenSSL Release]: https://github.com/openssl/tools/blob/master/HOWTO-make-a-release.md
+
+[^1]: Feature Acceptance Criteria - 🚧 The document is a work in progress.
+[^2]: Release Steering Committee - A group comprised of four individuals: one
+    internal member from OpenSSL, two specially invited representatives from the
+    community, and the engineering manager. This committee is dedicated to
```

Comment:
What is an internal member? Who can represent the community? Who determines that?